### PR TITLE
BAD-176 - Compile CDH 5.3 Shim

### PR DESCRIPTION
**Pull request by mtseu**
this is simply a copy of cdh52 with package renaming.
build.xml has been changed sinse cdh53 is to replace cdh52 shim.
build.properties has been changed to reference 5.3 Cloudera jars.

@mattyb149 @brosander could you please review/merge?
@lgrill-pentaho keeping you in the loop since there are some items to be done for build process for new shim
@mbradbury2014 FYI :)

### CODE REVIEW

**Comment by mtseu** [comment]
EE part of the CDH53 shim is formed as the following PR:https://github.com/pentaho/pentaho-big-data-ee/pull/57

**Comment by mtseu** [comment]
big data plugin PR: https://github.com/pentaho/big-data-plugin/pull/323

File: cdh53/build.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--===========================================================================
+  This is the build file for the CDH4 Shim modules.
+
+  See ../build-res/subfloor.xml for more details
+============================================================================-->
+<project name="pentaho-hadoop-shims-cdh52" basedir="." default="dist"
```
Comment (mattyb149) [inline cdh53/build.xml]: The project should be pentaho-hadoop-shims-cdh52, also you might as well update line 2 to say CDH 5.3 or something...

**Comment by mtseu** [comment]
@mattyb149  you are so right - fixed

**Pull request merged**